src/cerebral.d.ts

```diff
@@ -0,0 +1,107 @@
+﻿declare module "cerebral" {
```

Comment:
As a long as it'll work. I'm very sceptical that I'll be able to change the signature of a method by importing a types. Eg. Set on state provider is defined like

```
set(path: string, value: any)
```
Not

```
set<T>(path: T, value: T)
```

I want nothing to do with the standard types in my cerebral projects. No point using typescript then.